src/features/account/AccountActionBar.tsx

```diff
@@ -52,18 +51,14 @@ const AccountActionBar = ({
   const anim = useRef(new Animated.Value(1))
   const { currentAccount } = useAccountStorage()
 
-  const { data: voteData } = useVotesQuery({
-    variables: { address: currentAccount?.address || '' },
-    skip: !currentAccount?.address,
-    fetchPolicy: 'cache-and-network',
-  })
+  const voteData = useMemo(() => {
+    // TODO: Are votes still a thing?
```

Comment:
I dont think so haha? For another PR but we can probably take all the old vote stuff out..

Reply:
Yeah we can remove that